### ISSUE TITLE
1.17 Server connection problems 

### ISSUE THREAD

**Issue by Virtualfg** [minecraft-linux/mcpelauncher-manifest#541]
recently i added a new profile to mclauncher in a different directory in the hopes of signing in on a different account. After 2 hours of trying to sign in i did it and everything after that point went to hell. Firstly, when I launched minecraft it would take a century to load in my account and not only with the new profile but with my other profile too. Sometimes if I was luck enough to get the darn thing to sign in my account in less than 20 minutes, I could try to join server, Lifeboat for example. The moment i clicked ''Join Server'' and the UI popped up saying ''connecting to external server'' that was the only thing i would be looking at for the 1:30 minutes. It wouldn't join at all, i tried so many times, i thought it was some kind of luck based thing but no, not even patience worked. I went, i opened minecraft 8 times i had 8 windows running, trying to see which one will log in faster and which one with join the server. None of them joined. I panicked i deleted the new profile, yet still nothing. I tried everything, i deleted the launcher and reinstalled it. I left only the default profile. Nothing. Log in time are still horrible, and i cant join any servers. Realms don't even show up and when i click join server it doesn't even log it in the game, it doesn't even try to join. This has made my weekend hell and i seriously need to fix this.

**Comment by Virtualfg** [comment]
Update: I have deleted everything and set it up from the start 
still nothing, it says fetching realms in the friends category forever, it never fetches them same thing with the marketplace and realm invites. This seems to have a connection with my server problem just seems like its has no internet access or a really bad connection.

**Comment by Fich420** [comment]
Yeah, I have a LAN server and am having the same issue, on native windows launcher its there, but with this app it is just gone.

**Comment by Fich420** [comment]
> Yeah, I have a LAN server and am having the same issue, on native windows launcher its there, but with this app it is just gone.

Is now randomly working again for me. Are you still having issues @Virtualfg ?

**Comment by Virtualfg** [comment]
> > Yeah, I have a LAN server and am having the same issue, on native windows launcher its there, but with this app it is just gone.
> 
> Is now randomly working again for me. Are you still having issues @Virtualfg ?

for some random reason yes, it has even developed a pattern. You open Minecraft, wait till your skin starts to forever load, click play and wait for realms to show up then you can join a server. I tested and I was able to add another account, right know I have like 4 accounts connected so yeah it has weirdly worked out for some reason.You never know what will happen with this launcher.

**Comment by Fich420** [comment]
> > > Yeah, I have a LAN server and am having the same issue, on native windows launcher its there, but with this app it is just gone.
> > 
> > 
> > Is now randomly working again for me. Are you still having issues @Virtualfg ?
> 
> for some random reason yes, it has even developed a pattern. You open Minecraft, wait till your skin starts to forever load, click play and wait for realms to show up then you can join a server. I tested and I was able to add another account, right know I have like 4 accounts connected so yeah it has weirdly worked out for some reason.You never know what will happen with this launcher.

interesting, i havent had the profile issue you're having. at least we can play servers now though lol
